apis/fluent-en_GB/extensions/kotlin_1_3/atrium-api-fluent-en_GB-kotlin_1_3-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/kotlin_1_3/ResultFeatureAssertionsSpec.kt

```diff
@@ -10,17 +10,21 @@ class ResultFeatureAssertionsSpec : ResultFeatureAssertionsSpec(
     feature0<Result<Int>, Int>(Expect<Result<Int>>::isSuccess),
     fun1<Result<Int>, Expect<Int>.() -> Unit>(Expect<Result<Int>>::isSuccess),
     feature0<Result<Int?>, Int?>(Expect<Result<Int?>>::isSuccess),
-    fun1<Result<Int?>, Expect<Int?>.() -> Unit>(Expect<Result<Int?>>::isSuccess)
+    fun1<Result<Int?>, Expect<Int?>.() -> Unit>(Expect<Result<Int?>>::isSuccess),
+    "isFailure" to Companion::isFailureFeature
 ) {
     companion object {
-        @Suppress("unused", "UNUSED_VALUE")
+        private fun isFailureFeature(expect: Expect<Result<Int>>) = expect.isFailure<IllegalArgumentException>()
+
+        @Suppress("unused", "UNUSED_VALUE", "UNUSED_VARIABLE")
         private fun ambiguityTest() {
             var a1: Expect<Result<Int>> = notImplemented()
             var a2: Expect<out Result<Int>> = notImplemented()
             var a1b: Expect<Result<Int>> = notImplemented()
```

Comment:
has nothing to do with your change, but this should actually be
```suggestion
             var a1b: Expect<Result<Int?>> = notImplemented()
```
Would you mind to change that as well?
Same for the next line, it should be a `Expect<out Result<Int?>>`

Reply:
thanks :slightly_smiling_face: